packages/core/src/MPEContract.js

```diff
@@ -62,108 +64,121 @@ class MPEContract {
    * @returns {Promise.<TransactionReceipt>}
    */
   async withdraw(account, amountInCogs) {
-    logger.info(`Withdrawing ${amountInCogs}cogs from MPE account`, { tags: ['MPE'] });
+    const amount = new BigNumber(amountInCogs).toFixed();
```

Comment:
Extract into a helper method.

Reply:
Address in [commit](https://github.com/singnet/snet-sdk-js/pull/17/commits/5c2daa487965be1991c316051f66ccc837c9000a)